main.py

```diff
@@ -1423,8 +1441,10 @@ def handle_interruption_start(self, event: KytosEvent):
         )
         # for switch_id in switches:
         #     pass
-        # for interface_id in interfaces:
-        #     pass
+        for interface_id in interfaces:
+            interface = self.controller.get_interface_by_id(interface_id)
+            if interface:
+                self.notify_interface_status(interface, 'down', 'maintenance')
```

Comment:
@Alopalao this ended up hardcoding `maintenance`, we should pass the `interrupt_type` variable, see what I mean? That way any other future interruption type will be passed down correctly too.

Reply:
Changed